### ISSUE TITLE
Ocean model output in support of Denman Terrestrial Campaign

### ISSUE THREAD

**Issue by mmr0** [PaulSpence/Denman2024Voyage#18]
### What do we expect to see at our Shackleton sampling/mooring site?

Wish list (in rough priority order)
-  South/North and East/West transects of T & S intersecting (-66.2078, 98.8669), roughly 20km long to cover prospective hot water drill (HWD) sites
-  Current speed and its at (-66.2078, 98.8669) and its variability including tides. Variation over depth also of interest. 
-  TS plots from inside the Denman & Shackleton cavities
-  TS plots from outside the cavity - ideally focussing on watermasses that might make it into the cavity, e.g. the 500m isobath? Or bottom T & S on the shelf?
-  An estimate of the Cavity residence time for Shackleton and/or Denman?

When? Campaign is early Jan, so summer data is most relevant. Would be interested to see summer vs winter perhaps?

Brief summary of the Hot Water Drilling:
One borehole through the Shackleton Ice Shelf, near the western flank of the Denman Ice Shelf.  Ice approx 200m thick, water column approx total depth approx 600m, TBC. Data sampling will include CTD profiling, water sample collection, video logging and sediment collection. Following profiling, we will deploy a mooring with 2 x CTD (one near ice-ocean interface and one near seafloor, sampling @ 10 minute intervals) and 1 x current meter (near ice-ocean interface, sampling @ 30 minute intervals), and a distributed temperature sensing (DTS) fibre optic cable which will give meter-resolution temperature through both the ice and water column. Expected endurance of the ocean instruments is ~ 3 years. DTS data will be 1 year. At the surface, an APRES will collect basal melt rate data. 

### What do we expect to see near Duanne's epishelf lake sites?

Wish list:
- Summertime T & S transect from (-66.0987, 99.1761) to (-65.4753, 102.1599)
- Summertime T & S transect from (-65.413, 99.5659) to (-66.387, 100.453)

Any help on any of these would be amazing!!

**Comment by mmr0** [comment]
Box 1
lats = array([-65.4987, -65.7091, -65.6935, -65.4519])
lons = array([97.8152, 97.8152, 98.2762, 98.2574])

Box 2 
lats = array([-65.3273, -65.5688, -65.561 , -65.2961])
lons = array([98.5867, 98.6337, 99.1606, 99.123 ])

**Comment by claireyung** [comment]
### Some ACCESS-OM2-01 IAF Cycle 4 T-S Diagrams
Map:
![image](https://github.com/PaulSpence/Denman2024Voyage/assets/61528379/3337255f-3a47-4c6a-879f-af98f4a2d2ba)

T-S over 2000-2018 (monthly data):
![image](https://github.com/PaulSpence/Denman2024Voyage/assets/61528379/4276abf8-d84b-4dee-bb39-facaf0925499)

Only Januarys:
![image](https://github.com/PaulSpence/Denman2024Voyage/assets/61528379/d1318ae1-1efd-4675-95ae-664fe9dbf8f8)

Looks like some CDW coming in in Box2 but not in Box1?

**Comment by yuhang559** [comment]
**MITgcm, RYF (1990-91), BedMachine bathymetry**
Map: 
![Denman region campain](https://github.com/PaulSpence/Denman2024Voyage/assets/106302076/2db6fefd-f51b-4c1f-af4b-a5f4554080e2)
TS diagrams: 
![Jan box1](https://github.com/PaulSpence/Denman2024Voyage/assets/106302076/4c45666a-1003-4b7e-b96f-9f05b5e87187)
![Jan box2](https://github.com/PaulSpence/Denman2024Voyage/assets/106302076/15d4e736-4b59-4971-b03a-341b7a295015)
Looks water masses in MITgcm are much colder than ACCESS :)

**Comment by mmr0** [comment]
Yiew, thanks @claireyung @yuhang559, this is awesome! Yeah it looks like Boxes 1 and 2 are pretty similar except for Box 1 being much shallower and excluding CDW in ACCESS and mostly excluding it in MITgcm. We definitely didn't catch any ISW haha. How do I interpret all this near-freezing point fresher water in MITgcm e.g. in Box 2 in Jan? is this freshwater from melted sea ice?

**Comment by mmr0** [comment]
Ribiero et al 2023 - Change in watermass properties east of Shackleton from historical  -> recent data
https://agupubs.onlinelibrary.wiley.com/doi/pdf/10.1029/2023JC019882

Van Wijk 2022 - CDW observed in trough in front of/adjacent to Denman Ice Tongue from profiling floats
https://agupubs.onlinelibrary.wiley.com/doi/epdf/10.1029/2022GL100460

**Comment by yuhang559** [comment]
> Yiew, thanks @claireyung @yuhang559, this is awesome! Yeah it looks like Boxes 1 and 2 are pretty similar except for Box 1 being much shallower and excluding CDW in ACCESS and mostly excluding it in MITgcm. We definitely didn't catch any ISW haha. How do I interpret all this near-freezing point fresher water in MITgcm e.g. in Box 2 in Jan? is this freshwater from melted sea ice?

Yes, the freshwater in MITgcm is both from the sea ice and the melting of the ice shelf.

**Comment by yuhang559** [comment]
**MITgcm(RYF, tides) Current speed and two summer sections**

**Velocity map:** 
![fig1](https://github.com/PaulSpence/Denman2024Voyage/assets/106302076/b24bd807-37c7-43a1-98f3-699811b89611)

**Velocity-depth figure** (white circle in velocity map): Our model includes the tides, but we only save the monthly mean data, so it probably can't capture the tide variability and the velocity-depth figure shows the velocity is highly sensitive to the position of the dots. 
![1112vbox14](https://github.com/PaulSpence/Denman2024Voyage/assets/106302076/6f81fa46-aa42-4990-b53a-7ee6b348f364)

**TS diagram sections:** To plot it conveniently, I interpolated the matrix, so section 1 and section 2 look in different resolutions. 
![TSsection](https://github.com/PaulSpence/Denman2024Voyage/assets/106302076/8542e1e4-ae5a-4eb2-a7d2-d554612c6169)

**Comment by mmr0** [comment]
Thanks so much Yuhang, this is super useful! 

> **MITgcm(RYF, tides) Current speed and two summer sections**
> 
> **Velocity map:** ![fig1](https://user-images.githubusercontent.com/106302076/282657106-b24bd807-37c7-43a1-98f3-699811b89611.jpg)
> 
> **Velocity-depth figure** (white circle in velocity map): Our model includes the tides, but we only save the monthly mean data, so it probably can't capture the tide variability and the velocity-depth figure shows the velocity is highly sensitive to the position of the dots. ![1112vbox14](https://user-images.githubusercontent.com/106302076/282657310-6f81fa46-aa42-4990-b53a-7ee6b348f364.png)
> 
> **TS diagram sections:** To plot it conveniently, I interpolated the matrix, so section 1 and section 2 look in different resolutions. ![TSsection](https://user-images.githubusercontent.com/106302076/282658322-8542e1e4-ae5a-4eb2-a7d2-d554612c6169.png)

**Comment by dgwyther** [comment]
Is there still interest in these results? I have some ROMS Shackleton results that I could extract similar analyses for? @mmr0 ?
